src/screens/DiscoverScreen.tsx

```diff
@@ -1,30 +1,233 @@
-import { ShowCard } from '../components';
-import { useProfileContext, useTrendingShows } from '../hooks';
+import { useState, useEffect } from 'react';
+import { useTrendingShows } from '../hooks';
+import { Typography as Typ } from '@mui/material';
+import { ShowCarousel, Banner } from '../components';
+import { ShowData, DiscoverMovie, DiscoverTv } from '../types/tmdb';
+import { getDiscoverMovies, getDiscoverTv } from '../helpers';
 /**
- * Requests trending movies, passing data to ShowCard components.
+ * Requests a variety of filtered shows, rendering carousels
  * @returns {JSX.Element}
  */
 export default function DiscoverScreen(): JSX.Element {
```

Comment:
```suggestion
 * Requests a variety of filtered shows, rendering carousels
 */
const DiscoverScreen: React.FC = () => {
```
I'd like to try and standardize all component/screen function declarations like this.